### PR TITLE
tmux sessionaizer: make it more reliable 

### PR DESCRIPTION
**Pull request by saccarosium**
I apported some modification in order to make the script much more reliable. 
When I installed the script, it broke every time I was outside of tmux and I was trying to connect to an existing session.

Hope you find it useful

### CODE REVIEW

File: bin/.local/bin/tmux-sessionizer [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
+    # Adding support for simply pass the current working directory or a specified path
+    selected="$1" && [[ "$selected" == '.' ]] && selected="$PWD"
 else
     selected=$(find ~/work/builds ~/ ~/work ~/personal ~/personal/yt -mindepth 1 -maxdepth 1 -type d | fzf)
 fi
 
-if [[ -z $selected ]]; then
-    exit 0
-fi
+# This is work because '&&' do the action only if what came before is true.
```
Comment (ThePrimeagen) [inline bin/.local/bin/tmux-sessionizer:10]: stylistic changes often can cause slow down in prs.  could we drop this? i am just not going to accept it :)

I like the explicit if statement.

**Comment by ThePrimeagen** [comment]
i want to merge this, no need to close.

**Comment by saccarosium** [comment]
Sorry I have miss interpreted

File: bin/.local/bin/tmux-sessionizer (hunk 1 above)
Comment (saccarosium) [reply bin/.local/bin/tmux-sessionizer:10]: Is personal taste. But usually in shell scripting is faster doing a stream instead of a if statement.

**Comment by saccarosium** [comment]
Do you prefer with the truncated name?

If you have a very long name, for example "verylongdirectoryname", the resulting tmux name is:
Screenshot 2022-04-08 at 17 31 59

Applying the the modification in # PERSONAL PREFERENCE section the name become:
Screenshot 2022-04-08 at 17 33 24

I thinks it makes the name is more readable this way. It make a treashold of 8 characters, you can modify it as you wish (you can only make it smaller though).

**Comment by ThePrimeagen** [comment]
either or.  You pick :)

I like the truncated name

@saccarosium i just merged because i like this change so much (the functionality of it)

**Comment by ThePrimeagen** [comment]
Also, this seems to have broken C-a l (jump to previous session)

seems to open up two sessions.  I am going to revert this change and if you want another PR i'll be glad to merge :)

**Comment by ThePrimeagen** [comment]
also, gl have fun, i rewrote history.  the merge was giving me grief getting it out correctly.

**Comment by saccarosium** [comment]
I've open another PR

**Pull request merged**